framework/DataObjects/XDataSet.py

```diff
@@ -754,7 +756,7 @@ def _checkAlignedIndexes(self,rlz,tol=1e-15):
             # it's already gone; this can happen if this pivot parameter is only being used to collapse data (like in PointSet case)
             pass
         # otherwise, you're misaligned, and have been since before this realization, no action.
-      return
+    return
```

Comment:
Yes, this is a good fix.  I think we've only tested this with single indexes, so we would have missed this difference.